hazelcast-client/pom.xml

```diff
@@ -33,7 +33,6 @@
         <!-- needed for checkstyle/findbugs -->
         <main.basedir>${project.parent.basedir}</main.basedir>
         <atomikos-version>3.9.2</atomikos-version>
-        <maven.test.skip>true</maven.test.skip>
```

Comment:
fyi: by merging this PR we've re-enabled old client tests, which we should do anyway, since we will retain old client for EA